sopel/irc/__init__.py

```diff
@@ -179,29 +179,33 @@ def run(self, host, port=6667):
         try:
             self.backend.run_forever()
         except KeyboardInterrupt:
+            # raised only when the bot is not connected
             LOGGER.warning('Keyboard Interrupt')
-            self.quit('KeyboardInterrupt')
-
-    # Connection Events
+            raise
 
     def on_connect(self):
         """Handle successful establishment of IRC connection."""
+        LOGGER.info('Connected, initiating setup sequence')
+
         # Request list of server capabilities. IRCv3 servers will respond with
         # CAP * LS (which we handle in coretasks). v2 servers will respond with
         # 421 Unknown command, which we'll ignore
+        LOGGER.debug('Sending CAP request')
         self.backend.send_command('CAP', 'LS', '302')
 
         # authenticate account if needed
         if self.settings.core.auth_method == 'server':
+            LOGGER.debug('Sending serveur auth')
```

Comment:
Why serveur auth?

Should it be server?

Reply:
Pardon my French. :fr: :rofl:

Reply:
```suggestion
            LOGGER.debug('Sending server auth')
```

Reply:
You would think after studying a language for 7? Years I'd be able to reconigse it!